src/lib_ccaligner/params.cpp

```diff
@@ -11,7 +11,7 @@ const std::string currentTime()
     time_t now = time(0);
     struct tm tstruct = * localtime(&now);
     std::string localTime;
-    localTime.resize(19);   //len(%d-%m-%Y-%H-%M-%S)
+    localTime.resize(32);   //len(%d-%m-%Y-%H-%M-%S)
```

Comment:
Any specific reason, why this was changed? The string length is exact 19 characters?

It leads to 

```
logPath             : tempFiles/02-12-2017-11-49-42.log
phonemeLogPath      : tempFiles/phoneme-02-12-2017-11-49-42.log
```

getting changed to : 

```
logPath             : tempFiles/02-12-2017-11-52-03             .log
phonemeLogPath      : tempFiles/phoneme-02-12-2017-11-52-03             .log
```

Reply:
@saurabhshri I don't know why it works in Linux, but you need at least 19+1(\0)==20 bytes to store the result.

Reply:
@harrynull C++ strings do not need to end in NUL character :) 

Unlike character array, which is simply an array of characters terminated by a NUL character, a string is a class which defines objects that be represented as stream of characters. 

See the "String Length and Accessing Individual Elements" section at https://www.cprogramming.com/tutorial/string.html